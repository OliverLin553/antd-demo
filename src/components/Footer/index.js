import React from 'react';
import styles from "./style.module.css";

export default function Footer() {
  const date = new Date().getFullYear();

  return (
    <div className={styles.Footer}>
      <div className={styles.Container}>
        Â©2019-{date} Olin.
      </div>
    </div>
  );
}
