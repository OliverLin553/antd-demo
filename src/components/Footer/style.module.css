.Footer {
  height: 80px;
  line-height: 80px;
}

.Container {
  max-width: 1280px;
  margin: 0 auto;
  border-top: 1px solid #f0f0f0;
  text-align: right;
}
