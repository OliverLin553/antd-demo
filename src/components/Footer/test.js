import React from 'react';
import { shallow } from 'enzyme';
import Footer from './';

describe('<Footer>', () => {
  it('it renders the expected tree', () => {
    const wrapper = shallow(<Footer />)
    expect(wrapper).toMatchSnapshot()
  })
})
