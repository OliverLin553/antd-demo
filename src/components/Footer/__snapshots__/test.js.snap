// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer> it renders the expected tree 1`] = `
<div
  className="App"
>
  123
</div>
`;
